ursabot/builders.py

```diff
@@ -171,12 +171,14 @@ def _render_properties(self):
         props = Properties(
             buildername=str(self.name),
             builddir=str(self.builddir),
-            workerbuilddir=str(self.workerbuilddir)
+            workerbuilddir=str(self.workerbuilddir),
+            docker_image=str(self.image),
+            docker_workdir=self.image.workdir,
```

Comment:
Volumes are mounted from the CLI using the `docker_workdir` property in the interpolation, but it wasn't passed to the rendering function.